LeanplumTests/LeanplumTests.m

```diff
@@ -136,7 +136,7 @@ - (void) testUserAttributesError
     [[LPRequestQueue sharedInstance] sendRequests:^{
         NSLog(@"test");
     } failure:^(NSError * _Nonnull error) {
-        NSLog(@"failrure");
+        NSLog(@"failure");
```

Comment:
could these failure messages be more descriptive?

Reply:
These are test cases. But yea, we can basically log the error. I will do that as part of the other PR, cos this affects almost every where we wrote tests.